app/javascript/components/aggregate_status_card.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { Quadicon } from '@manageiq/react-ui-components/dist/quadicon';
+
+import PfAggregateStatusCard from './pf_aggregate_status_card';
+import { http } from '../http_api/';
+
+const AggregateStatusCard = ({ providerId, providerType }) => {
+  const [data, setCardData] = useState({ loading: true });
+
+  useEffect(() => {
+    const url = `/${providerType}_dashboard/aggregate_status_data/${providerId || ''}`;
+    http.get(url)
+      .then((response) => {
+        console.log(response.data);
```

Comment:
@martinpovolny the `console.log()` is left there intentionally?

Reply:
No. Thanks!